lib/jsonApi.js

```diff
@@ -37,6 +38,7 @@ jsonApi._cleanBaseUrl = function(base) {
 
 jsonApi.define = function(resourceConfig) {
   resourceConfig.namespace = resourceConfig.namespace || "default";
+  resourceConfig.handlers = resourceConfig.handlers || jsonApi.defaultHandlers;
```

Comment:
Can we drop the addition on line 15 and tweak this line to simply `||` against `mockHandlers.handlers`? I can't see any future use for duplicating the property above?